README.md

```diff
@@ -1 +1,297 @@
+
+
+#### Selecting a Node
+
+When clicking a node, this node will be selected and the info will be visible in the top left corner.
+
+![select node](images/select_node.png)
+
+The image shows the info of the selected node. There are two more buttons showed as well. These two buttons pop up when a node is selected. These will be explained in [highlight connecting nodes](####highlight_connecting_nodes) and [Add/show connecting nodes](####add/show_connecting_nodes).
+
+#### Hovering a Node
+
+When hovering a nodes, the name of that node is shown.
+In the previous image, the black node at the bottom was hovered by the mouse.
+
+#### HTML link in Info of Node
+
+When `html_documentation_root` is specified, the name of the node will be a link to the documentation of that specific node.
+
+#### Highlight Connected Nodes
+
+When the button with the eye is clicked, the nodes connecting to the selected node will be highlighted. The rest of the nodes get a lower opacity.
+
+![highlight connected nodes](images/highlight_connected_nodes.png)
+
+
+#### Add/Show Connecting Nodes
+
+When the button **Add/show connecting nodes** is clicked, the nodes connecting to the selected node will be obtained. It will add new nodes or links, if there are any.
+
+Before pushing the button:
+![before adding](images/before_adding.png)
+
+After pushing the button:
+![add connecting nodes](images/add_connecting_nodes.png)
+
+#### Node labels
+
+The labels/names of the nodes are shown when the checkbox of **Nodes with labels** is active.
+
+![show labels](images/show_labels.png)
+
+#### Zoom buttons
+
+There are three buttons for zooming, respectively zoom in, zoom out and zoom till the network fits the canvas.
+
+![zoom buttons](images/zoom.png)
+
+#### Errors
+
+When an error occurs you get a notification at the top of the page as shown in following example where an invalid Cypher query is submitted:
+
+![error](images/error.png)
+
+You can show the error by clicking **Show error message**:
+
+![error](images/show_error.png)
+
+You can close the error by clicking **X**.
+If the error is not closed and another error occurs, all errors will be listed at the top of the page.
+You will be redirected to the hight of the newest error.
+
+## Bugs
+
+1. When showing the labels/names of the nodes, there is allways one node that turns black.
+1. When using the input fields, the autocompletion list will always look at the last part of the sentence, so when adding something in the middle the autocompletion will stay the same (focused on the last part).
+
+
+
+## Features
+
+- Errors could be popping up, you need to close it before you can go further. That way the user can't miss it.
+- Faster rendering; open to suggestions
+- Instead of disabling the button when submitted, stop the previous request when another submit is done.
+- A node that needs a layer but the group is not specified, better to place it on the same hight as the first node that is connected to it.
```

Comment:
```suggestion
- A node that needs a layer but the group is not specified, better to place it on the same height
  as the first node that is connected to it.
```
Please rephrase this sentence.

Reply:
Resolved in 3bc1af842099640e02bb7e825ab4b318196169b1